ddtrace/llmobs/_llmobs.py

```diff
@@ -451,6 +453,22 @@ def _llmobs_tags(span: Span, ml_app: str, session_id: Optional[str] = None) -> L
         existing_tags = span._get_ctx_item(TAGS)
         if existing_tags is not None:
             tags.update(existing_tags)
+
+        # set experiment tags on children spans if the tags do not already exist
+        experiment_id = span.context.get_baggage_item(EXPERIMENT_ID_KEY)
+        if experiment_id:
+            # the children spans of an experiment span should be tagged by the experiment ID as well
+            if "experiment_id" not in tags:
+                tags["experiment_id"] = experiment_id
+
+        run_id = span.context.get_baggage_item(EXPERIMENT_RUN_ID_KEY)
+        if run_id and "run_id" not in tags:
+            tags["run_id"] = run_id
+
+        run_iteration = span.context.get_baggage_item(EXPERIMENT_RUN_ITERATION_KEY)
```

Comment:
what's the purpose of propagating experiment run id/iteration to the downstream service? do we need this tag on all spans in the trace?

Reply:
yea, we need to have all all spans in the trace identifiable and tied to the experiment so that backend can optimize loading time